php-fpm/recipes/configure.rb

```diff
@@ -52,7 +50,13 @@
   Chef::Log.info("Adding to php sendmail stmt: #{sendmail_params}")
 end
 
-template "#{etc_fpm_dir}/#{conf_fpm}" do
+execute 'update-alternatives' do
+  command '/usr/bin/update-alternatives --install /usr/sbin/php-fpm php-fpm /usr/sbin/php-fpm5.6 0'
+  creates '/usr/sbin/php-fpm'
```

Comment:
@fh I pushed 2 commits addressing this and:

 1. Ensuring `php-fpm5.6`, but also `php-fpm7.0` work
 2. Wrote a test for that (assert on the `update-alternatives` command)